src/txn/txn.c

```diff
@@ -1231,8 +1231,20 @@ __txn_resolve_prepared_op(WT_SESSION_IMPL *session, WT_TXN_OP *op, bool commit,
       (upd->type != WT_UPDATE_TOMBSTONE ||
         (upd->next != NULL && upd->durable_ts == upd->next->durable_ts &&
           upd->txnid == upd->next->txnid && upd->start_ts == upd->next->start_ts));
-    first_committed_upd_in_hs =
-      first_committed_upd != NULL && F_ISSET(first_committed_upd, WT_UPDATE_HS);
+    /*
+     * If the first committed update after the prepared update has already been marked to be
+     * deleted from the history store, we are in the case that there was a previous prepared update
```

Comment:
same here:
`we are in the case that there was a **next** prepared update . . .`

Reply:
I don't understand what you mean by next?

Reply:
I will resolve this as the updated comment makes sense.